Wikipedia.xcodeproj/xcshareddata/xcschemes/Wikipedia.xcscheme

```diff
@@ -92,7 +92,8 @@
       ignoresPersistentStateOnLaunch = "NO"
       debugDocumentVersioning = "YES"
       debugServiceExtension = "internal"
-      allowLocationSimulation = "YES">
+      allowLocationSimulation = "YES"
```

Comment:
The next commit will make the flipping happen automatically and this line will be removed. I just left it in case you wanted to check this commit out for testing so it would work as-is.